### ISSUE TITLE
Multi-level Classification?

### ISSUE THREAD

**Issue by ArchithaKrishna** [quark0/darts#24]
Works with single level classification, however does this support images with multilevel classification, if so could you help me with the classifying my data set, with images more than 60 categories?

**Comment by quark0** [comment]
This could be done by modifying the single-label cross-entropy loss functions specified in `train.py` and `train_search.py`. Sorry but I can't provide more help for specific use cases.
